### DISCUSSION TITLE
docker no version 3.8.35

### DISCUSSION THREAD

**Discussion by ahgaoyong** [rabbitmq/rabbitmq-server#5686] [Other]
docker no version 3.8.35

**Comment by michaelklishin** [comment]
[RabbitMQ 3.8 has reached out of life](https://www.rabbitmq.com/versions.html) a full month ago, so my guess is that the [Docker image maintainers](https://github.com/docker-library/rabbitmq/) decided to not promote `3.8.35`.

Which is fine with me. Please switch to `3.10.7`.
